docs/Operators.md

```diff
@@ -1848,6 +1848,8 @@ Other versions of this operator: <a href="Changelog.md#BatchNormalization-1">Bat
 <dd>running (training) or estimated (testing) mean tensor of shape (C).</dd>
 <dt><tt>var</tt> : T</dt>
 <dd>running (training) or estimated (testing) variance tensor of shape (C).</dd>
+<dt><tt>training_mode</tt> (optional) : T1</dt>
```

Comment:
>training_mode [](start = 8, length = 13)

great! This flag has to be an input, instead of an attribute, since we need to modify it during the runtime. 

Please add this to the PR description.